mstrio/utils/dtos/create_attribute_dto.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class CreateAttributeDto:
+    """ DTO for creating an attribute
+
+    Args:
+        bodies: List of attribute creation data
+        show_expression_as: Specifies the format in which the expressions are
+           returned in response
+           Available values: 'tokens', 'tree'
+           If omitted, the expression is returned in 'text' format
+           If 'tree', the expression is returned in 'text' and 'tree' formats.
+           If 'tokens', the expression is returned in 'text' and 'tokens'
+           formats.
+        show_potential_tables: Specifies whether to return the potential tables
```

Comment:
do you expect the show_potential_tables to have other values besides boolean?

Reply:
good question, unfortunately I just copy-pasted what MSTR did